src/features/views/components/ViewColumnDialog/ColumnChoiceCard.tsx

```diff
@@ -0,0 +1,77 @@
+import { FormattedMessage as Msg } from 'react-intl';
+import { Box, Button, Card, Typography } from '@mui/material';
+
+interface ColumnChoiceCardProps {
+  addButtonLabel?: string;
+  alreadyInView?: boolean;
+  cardVisual: JSX.Element;
+  description: string;
+  color: string;
+  configureButtonLabel?: string;
+  onAdd: () => void;
+  onConfigure: () => void;
+  showAddButton?: boolean;
+  showConfigureButton?: boolean;
+  title: string;
+}
+
+const ColumnChoiceCard = ({
+  addButtonLabel = 'add',
```

Comment:
I foresee us wanting to refactor this, because right now there is no way for the user of this component to supply a fully custom label (unless that's label is stored in the "framework" part of the locale YAML file). But we'll cross that bridge if we get there.

Reply:
yeah it's not the most elegant, but like you say, we fix when we hit our heads in it